packages/hint-font-relative-units/src/_locales/en/messages.json

```
@@ -0,0 +1,14 @@
+{
+    "description": {
+        "description": "Metadata description",
+        "message": "Ensure font styles use relative units to enable user stylesheet overrides"
+    },
+    "name": {
+        "description": "Metadata name",
+        "message": "font-relative-units"
+    },
+    "pixelError": {
+        "description": "Metadata pixelError",
+        "message": "$1 should be defined with relative units instead of pixels"
```

For consistency with other messages can you please update this (and the tests then) to:

```suggestion
        "message": "$1 should be defined with relative units instead of pixels."
```